dev/Repeater/ItemsRepeater.idl

```diff
@@ -429,8 +429,11 @@ unsealed runtimeclass StackLayout : VirtualizingLayout
     [MUX_DEFAULT_VALUE("0.0")]
     Double Spacing { get; set; };
 
+    Boolean DisableVirtualization { get; set; };
```

Comment:
>DisableVirtualization [](start = 12, length = 21)

I suggest virtualmode or IsVirtualized or something. .net has virtualmode for listview.
[https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.listview.virtualmode?view=netframework-4.8](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.listview.virtualmode?view=netframework-4.8)

Reply:
If it's virtualmode, I prefer to use enum

---
In reply to: [352742192](https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r352742192) [](ancestors = 352742192)

Reply:
As Ranjesh pointed out, this example of VirtualMode is not exactly what we are trying to do here with StackLayout. I'll merge as is for now since API is still in preview, however this will be a good thing to discuss when this goes through API review.